### PR TITLE
Find orbiting flights in subset of data

### PR DESCRIPTION
**Pull request by kmcinerny**

### CODE REVIEW

File: summary-stats/find-orbits.R [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+##   data frame (data.frame, tbl)
+###     A data frame for the original data with five additional variables:
+####      next_heading (numeric): The heading of the next timestamp (˚)
+####      direction (numeric): The difference in heading between the next and current timestamps (˚)
+####      total_rotate (numeric): The total rotation of the helicopter (cumulative sum of all prior timestamps in the flight, ˚)
+####      ever_orbiting (boolean/logical): Did the total rotation of the helicopter ever reach orbit_threshold?
+####      is_orbiting (boolean/logical): Is the helicopter as orbiting or not at the given timestamp for the given flight?
+
+determine_orbit_positions <- function(monthly_data,
+                                      minute_range = 30,
```
Comment (shapironick) [inline summary-stats/find-orbits.R:29]: Its probably intentional but I'm just trying to understand the default above being 15 mins but then the range is 30 mins? I'm probably missing something

Comment (kmcinerny) [reply summary-stats/find-orbits.R:29]: I changed it to 30 mins from the default. But I'm going to change it back because I missed the +/- 15 mins part.

Comment (kmcinerny) [reply summary-stats/find-orbits.R:29]: I did purposefully change it to 30 mins, but now I see the minute range of +/-15 adds up to 30 minutes.

Comment (shapironick) [reply summary-stats/find-orbits.R:29]: OK great! i wasn't totally clear on the +/- 15 mins so maybe you can walk me through it on Friday and we can also send it to John to get any further feedback. thank you!

**Pull request merged**